travis/scala_classpath_for_python.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# Compile classes used in python tests
+sbt mleap-spark-extension/compile
+sbt mleap-spark/compile
+
+# Export the complete dependencyClasspath into an env. variable
+export SCALA_CLASS_PATH=\
+"$(sbt --error 'set showSuccess := false' mleap-spark-extension/printClassPath 2> /dev/null)"
```

Comment:
Now if this needs to be run manually any way, there's no need for a custom sbt task at all.

This single command provided by sbt should do it:

`sbt "export mleap-spark-extension/runtime:fullClasspath"`

This is nice because this includes the paths to `classes` folders so you can edit scala code, let your IDE compile it, and rerun python tests without visiting sbt/command line.

Reply:
Oh, interesting, that's useful, I'll update that, thanks!